.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+# Write Typical python hooks: black, flake8, mypy, pylint, yapf
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
+    - id: trailing-whitespace
+    - id: end-of-file-fixer
+    - id: check-yaml
+    - id: debug-statements
+    - id: double-quote-string-fixer
+    - id: name-tests-test
+      args: [--unittest]
+    - id: requirements-txt-fixer
+
+-   repo: https://github.com/hhatto/autopep8
```

Comment:
We use `black` for style - so the `double-quote-string-fixer` will also have to go.